src/fe-small-advertisements/src/actions/name.ts

```diff
@@ -0,0 +1,16 @@
+export interface User {
+  name: string;
+}
+
+export const CHANGE_NAME = 'CHANGE_NAME';
+
+export interface ChangeNameAction {
+  type: typeof CHANGE_NAME;
+  payload: User;
+}
+
+export const changeNameActionCreator = ({
```

Comment:
Also I think here the name could be improved a little. You are changing the name indeed but actually you are setting the name. Something like `setName` might be more descriptive? 
This is only a suggestion, `changeName` is perhaps alright, so feel free to do as you please